src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -212,7 +212,7 @@ protected async Task<IActionResult> ExecuteQuery<TPostBodyResource>(TResource re
                         _assemblyCacheService.AddAssemblyToCache(tempAsm.FullName, tempAsm);
                     }
 
-                    Request.HttpContext.Response.Headers.Add("script-etag", Convert.ToBase64String(ScriptCompilation.GetHashFromScript(jsonBody.Script)));
+                    Request.HttpContext.Response.Headers.Add("diag-script-etag", Convert.ToBase64String(ScriptCompilation.GetHashFromScript(jsonBody.Script)));
```

Comment:
Are you not passing assembly name in response header ?

Reply:
It is passed in the response of compilation output.